backend/app/modules/user/antisybil/service/initial.py

```diff
@@ -31,12 +36,16 @@ def get_antisybil_status(
             )
             raise ex
         if score is not None:
```

Comment:
I think this logic should be done in the update service, not in get_antisybil_status. Otherwise we'll have inconsistent results in db with the factual state and it'll be a way harder to debug in case of any bugs / misalignements.

Reply:
Yeah, this should be applied on a higher level. Changing

Reply:
Doing this on write would require adding additional marker in the DB. I dislike adding more state.